index.html

```diff
@@ -297,61 +297,61 @@ <h3 class="categories__title">Categorías para ti </h3>
     <section class="functionality">
       <div class="functionality__information container">
         <div class="functionality__carousel-wrapper">
-          <div class="functionality__carousel js-functionality-carousel">
-            <div class="functionality__icons js-functionality-icons">
+          <ul class="functionality__carousel js-functionality-carousel">
+            <li class="functionality__icons js-functionality-icons">
               <div class="functionality__fund">
-                <div class="functionality__overlay"></div>
+                <div class="functionality__overlay" aria-hidden="true"></div>
```

Comment:
No es necesario usar este atributo. Los `div` como tal no aportan a la semantica, asi que no hace falta ignorarlos
```suggestion
                <div class="functionality__overlay"></div>
```